src/fsharp/FSharp.Core/async.fsi

```diff
@@ -282,6 +282,14 @@ namespace Microsoft.FSharp.Control
         /// and not infinite.</exception>
         static member Sleep: millisecondsDueTime:int -> Async<unit>
 
+        /// <summary>Creates an asynchronous computation that will sleep for the given time. This is scheduled
+        /// using a System.Threading.Timer object. The operation will not block operating system threads
+        /// for the duration of the wait.</summary>
+        /// <param name="dueTime">The amount of time to sleep.</param>
+        /// <returns>An asynchronous computation that will sleep for the given time.</returns>
+        /// <exception cref="System.ArgumentOutOfRangeException">Thrown when the due time is negative.</exception>
+        static member Sleep: dueTime:TimeSpan -> Async<unit>
```

Comment:
`[<Experimental(ExperimentalAttributeMessages.RequiresPreview)>]` is required here

Reply:
fixed.